WordPress/WordPressTest/ReferrerDetailsViewModelTests.swift

```diff
@@ -0,0 +1,78 @@
+import XCTest
+@testable import WordPress
+
+class ReferrerDetailsViewModelTests: XCTestCase {
+    var sut: ReferrerDetailsViewModel!
+
+    override func setUpWithError() throws {
```

Comment:
Just a question: Why use `setUpWithError` vs `setUp`?

Reply:
Well in this particular case this newer method brings no benefits, since nothing can fail and no errors can be thrown. It's just a habit.

Reply:
Makes sense, that's what I was thinking just wanted to check. 👍